### PR TITLE
simple Perl wrapper for measuring test loss and train loss

### PR DESCRIPTION
**Pull request by martinpopel**
Vowpal Wabbit reports progressive validation loss or holdout loss by default,
but sometimes we need more:
 - test/holdout loss on a given file (not on 1/10 of the training data)
   This may be useful e.g. when comparing different training data sizes,
   i.e. plotting the learning curve.
 - train loss (to assess the amount of overfitting)
 - automatic logging of experiments
 - brief results (only the overall losses) on stdout,
   details logged into a file

### CODE REVIEW

**Comment by martinpopel** [comment]
Travis failed on test 83 (diff in train-sets/ref/cb_adf.stderr), but this is clearly not related to this pull request. It is related to John's commit  4c0856437f784f6c0353db79c4318c0000ff508b see https://travis-ci.org/JohnLangford/vowpal_wabbit/builds/87769102

**Comment by JohnLangford** [comment]
My bad, fixed.

-John
